src/plugin/ipc/socket/socketconnection.cpp

```diff
@@ -221,6 +235,15 @@ bool TcpConnection::isBlacklistedTcp(const sockaddr* saddr, socklen_t len)
     int blacklistedRemotePorts[] = {53,                 // DNS Server
                                     389, 636,           // LDAP
                                     -1};
+#ifdef MPISPAWN_WORKAROUND
```

Comment:
May be a more readable design here would be to replace the int array with a vector and then just do a push_back to the vector inside this #ifdef. What do you think?